src/Components/NavBar/Menus/MoreNavMenu.tsx

```diff
@@ -1,23 +1,55 @@
 import { Menu, MenuItem } from "@artsy/palette"
-import React from "react"
+import React, { useContext } from "react"
+import { NavbarContext } from "../NavBar"
 
 export const MoreNavMenu: React.FC = () => {
+  const { tracking, Schema } = useContext(NavbarContext)
+
+  const trackClick = event => {
+    event.preventDefault()
+    const link = event.currentTarget
+    const text = link.innerText
+    const href = link.getAttribute("href")
+
+    tracking.trackEvent({
+      flow: Schema.Flow.Header,
+      context_module: Schema.ContextModule.HeaderUserDropdown,
+      subject: text,
+      destination_path: href,
+    })
+  }
+
   return (
     <Menu title="More">
       {/*
         Hide nav items at md / lg as they appear in the top nav
       */}
-      <MenuItem href="/galleries" display={["block", "block", "none"]}>
+      <MenuItem
+        href="/galleries"
+        display={["block", "block", "none"]}
+        onClick={trackClick}
```

Comment:
Rather than defining this prop on each individual item, can’t you rely on DOM events bubbling up the tree and define the handler once on the `Menu` component? The usage of `event.currentTarget` should iirc already provide you with the actual clicked link.

Reply:
Will try this 👍